### PR TITLE
Adding ks-installer kubesphere scenario

### PR DESCRIPTION
**Pull request by AbdelouahabMbarki**
i have added the scenario mentioned in this issue kubesphere/community#355

### CODE REVIEW

File: Install-kubesphere-on-kubernetes/index.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
+{
+  "title": "Install kubesphere on kubernetes",
```
Comment (LinuxSuRen) [inline Install-kubesphere-on-kubernetes/index.json:2]: `kubesphere` -> `KubeSphere`

Comment (LinuxSuRen) [reply Install-kubesphere-on-kubernetes/index.json:2]: `kubernetes` -> `Kubernetes`

File: README.md [hunk 2]
```diff
@@ -1,2 +1,5 @@
-# katacoda-scenarios
-kubesphere Katacoda Scenarios
+# Interactive kubesphere Katacoda Scenarios
```
Comment (LinuxSuRen) [inline README.md:1]: typo

File: README.md [hunk 3]
```diff
@@ -1,2 +1,5 @@
...
+[![](http://shields.katacoda.com/katacoda/kubesphere-sigs/count.svg)](https://www.katacoda.com/kubesphere-sigs "Get your profile on Katacoda.com")
+
+Visit https://www.katacoda.com/kubesphere-sigs to view the profile and interactive scenarios
```
Comment (LinuxSuRen) [inline README.md:4]: image

File: Install-kubesphere-on-kubernetes/step1.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+3.Check if there is a default Storage Class in your cluster.`kubectl get sc`{{execute}} 
+
+> No storage Class found in the cluster
+
+```bash
+controlplane $ kubectl get sc 
+No resources found in default namespace.
+```
+
+4.install openebs storage Class in the cluster
```
Comment (LinuxSuRen) [inline Install-kubesphere-on-kubernetes/step1.md:24]: Start with capital?

**Review by LinuxSuRen — CHANGES_REQUESTED** [review]
Great! Glad to see the awesome contribution. I left some comments. Please take a look at it.

File: README.md (hunk 3 above)
Comment (AbdelouahabMbarki) [reply README.md:4]: I think we should cretae a profile for the organization in katacoda 
Similar to what i did in my account

Comment (LinuxSuRen) [reply README.md:4]: Got you. Please go ahead, and help us to create a profile.

**Comment by AbdelouahabMbarki** [comment]
change made

**Review by LinuxSuRen — APPROVED** [review]
LGTM

**Comment by AbdelouahabMbarki** [comment]
if it's OK we can merge it so this profile https://www.katacoda.com/kubesphere can point on it

**Comment by AbdelouahabMbarki** [comment]
i added the  logo to the project in the last commit
image

**Review by FeynmanZhou — APPROVED** [review]

**Pull request merged**